src/main/views/live-case-status.njk

```diff
@@ -16,7 +16,8 @@
 {% block content %}
     <h2 class="govuk-heading-l">Live hearing updates</h2>
```

Comment:
somehow looks like this line has been switched back, do you mind replacing with locale placeholder? it will look like 
```suggestion
    <h2 class="govuk-heading-l">{{ heading }}</h2>
```

Reply:
Sorted.